### ISSUE TITLE
weight decay

### ISSUE THREAD

**Issue by bpwl0121** [facebookresearch/llama#958]
Hi,
thanks for your awesome  work!

I have questions regarding the weight decay. 
1. Why do you set the weight decay to 0.1 during the pretraining stage which is actually quite large?
2. why do you set the weight decay to 0 during the FT stage since you set it to 0.1 during the previous stage?

best
